eZ/Publish/API/Repository/Tests/PermissionResolverTest.php

```diff
@@ -1166,12 +1166,18 @@ public function testLookupLimitationsWithRoleLimitationsWithoutPolicyLimitations
         $permissionResolver->setCurrentUserReference($user);
         /* END: Use Case */
 
+        foreach ($role->getPolicies() as $policy) {
+            if ($policy->module === $module  && $policy->function === $function){
```

Comment:
In this case I would actually use `array_filter`. Then referring to `[0]` should give you what you expect. Please notice also that $expectedPolicy might be uninitialized.

Side: CS is failing.

Reply:
It is fixed, thanks